### PR TITLE
Refactor JS/CSS, some perf work

### PR DESCRIPTION
**Pull request by bf4**

### CODE REVIEW

File: _includes/themes/left/default.html [hunk 1]
```diff
@@ -1,23 +1,46 @@
 <!DOCTYPE html>
-<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en-us">
```
Comment (nateberkopec) [inline _includes/themes/left/default.html]: lol xhtml

File: _includes/themes/left/default.html [hunk 2]
```diff
@@ -1,23 +1,46 @@
...
+    <!-- Le styles -->
+    <link rel="stylesheet" href="{{ ASSET_PATH }}/bootstrap/css/bootstrap.2.2.2.min.css">
+    <link rel="stylesheet" href="{{ ASSET_PATH }}/css/style.css?body=1" type="text/css" media="all">
+    <link rel="stylesheet" href="{{ ASSET_PATH }}/css/base.css" type="text/css" media="screen, projection" />
+    <link rel="stylesheet" href="{{ ASSET_PATH }}/css/pygments.css" type="text/css" />
+    <link rel="stylesheet" href="{{ ASSET_PATH }}/css/print.css" type="text/css" media="only print" />
+    <link rel="stylesheet" href="{{ ASSET_PATH }}/css/iphone.css" type="text/css" media="only screen and (max-device-width: 480px)" />
+    <link rel="stylesheet" href="{{ ASSET_PATH }}/css/iphone.css" type="text/css" media="only screen and (device-width: 768px)" />
+    <link href="//fonts.gstatic.com" rel="preconnect" crossorigin>
+    <link rel="stylesheet" href="http://fonts.googleapis.com/css?family=Yanone+Kaffeesatz&text==NATE%20MAKES%20APPS%20FAST" type="text/css">
```
Comment (nateberkopec) [inline _includes/themes/left/default.html]: lol I don't think you want that &text option

File: pair.md [hunk 3]
```diff
@@ -20,3 +18,5 @@ I've found good luck pairing with [this script](https://gist.github.com/bf4/8324
 ## Pair and tell
 
 <div data-component='print-pair-data' data-attrs='{"config": { "key": "0AqHUOZcVEj_XdE5SMzBKSWhINjVtTlh2b0JjUFp4OEE/od6", "fields": ["appointments","link","pair","description"] } }'></div>
+<script type="text/javascript" src="{{ BASE_PATH }}/assets/vendor.js"></script>
+<script async type="text/javascript" src="{{ BASE_PATH }}/assets/viewtastic.js"></script>
```
Comment (nateberkopec) [inline pair.md]: why put this at the bottom?

File: pair.md [hunk 4]
```diff
@@ -5,9 +5,7 @@ date: 2013-06-19 01:07:00
 slug: pair
 title: Pair
 group: listed_pages
-javascripts:
-- "/assets/vendor.js"
-- "/assets/viewtastic.js"
+javascripts: []
 ---
 
 ## [![Pair Prograrm With Me](http://www.pairprogramwith.me/badge.png 'Pair Program With Me')](http://www.pairprogramwith.me/)
```
Comment (nateberkopec) [inline pair.md]: typo

File: _includes/themes/left/default.html (hunk 2 above)
Comment (bf4) [reply _includes/themes/left/default.html]: @nateberkopec Any reason why not? I don't have non-latin characters.

File: pair.md (hunk 3 above)
Comment (bf4) [reply pair.md]: It wasn't working when I made them both async in the head for some reason and I didn't have time to investigate per value of 'getting it right' :)

File: pair.md (hunk 4 above)
Comment (bf4) [reply pair.md]: Wow, that took me a couple of reads to find `s/Prograrm/Program` I wonder why my spell checker didn't catch this

File: _includes/themes/left/default.html (hunk 2 above)
Comment (nateberkopec) [reply _includes/themes/left/default.html]: I was referring to the value, not the key 😆

File: pair.md (hunk 3 above)
Comment (nateberkopec) [reply pair.md]: Hm. That means it probably only *kind of* works at the bottom. Async scripts can load at *any time*, putting it at the bottom may just be causing it to load *slightly later*, but that's very connection dependent. I would keep it synchronous until you figure it out.

File: _includes/themes/left/default.html (hunk 2 above)
Comment (bf4) [reply _includes/themes/left/default.html]: Ah, I considered using something else, but I thought I've give you credit and make it fun :)

**Pull request merged**